### PR TITLE
skip `Glm4MoeModelTest::test_torch_compile_for_training`

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

We have line

> token_indices, weight_indices = torch.where(mask)

which won't work with this test.

It's skipped also in dots1 and deepseek

### CODE REVIEW

**Review by Cyrilvallez — APPROVED** [review]
Alright, thanks!
